packages/elements/src/clock/index.ts

```diff
@@ -699,6 +689,24 @@ export class Clock extends ResponsiveElement {
     this.renderRoot.addEventListener('tapstart', (event) => this.onTapStart(event as TapEvent));
   }
 
+  /**
+   * Handles interactive by update role, tabindex, and aria attribute
+   * @returns {void}
+   */
+  private interactiveChanged (): void {
+    if (this.interactive) {
+      this.setAttribute('role', 'spinbutton');
+      this.setAttribute('tabindex', '0');
```

Comment:
Should not force set tabindex to `0` because user can set it more than `0`. You must check it is already exist first.

Reply:
Updated by check tabindex before assign